### DISCUSSION TITLE
CT’s for mains on 200A US service

### DISCUSSION THREAD

**Discussion by genebean** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#170] [Q&A] [answered]
I’m confused on what I need for the mains coming into my breaker box. I am in the US and have 200A service. Do I put a 120A clamp on each wire or does each need a 200A clamp? Whatever is needed is intended to plug into the 6ch board with one or two expansion boards attached.

**Comment by chcornell25649** [comment, marked as answer]
the 120 on each should be perfect. Even a 100 should work if you get the larger clamp size.

**Comment by TRSx80** [comment]
Further info:

[US 200 Amp Service explained or Do I need a 200 Amp CT?](https://community.openenergymonitor.org/t/us-200-amp-service-explained-or-do-i-need-a-200-amp-ct/4219)
